backend/src/utils/array.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Add all {@link elements} to {@link array}. This mutates {@link array} and returns it.
+ * @param array The array to append to
+ * @param elements The elements to append
+ * @returns The appended array
+ */
+export function pushAll<T>(array: T[], elements: T[]): T[] {
```

Comment:
```suggestion
export function pushAll<T>(array: T[], elements: T[]): void {
```

Change the return type to `void`, to make sure the programmer knows this mutates the original array instead of returning a new array.